src/halmos/__main__.py

```diff
@@ -237,6 +237,15 @@ def rendered_trace(context: CallContext) -> str:
         return output.getvalue()
 
 
+def rendered_calldata(calldata: List[Byte]) -> str:
+    if any(is_bv(x) for x in calldata):
+        # make sure every byte is wrapped
+        calldata_bv = [x if is_bv(x) else con(x, 8) for x in calldata]
+        return hexify(simplify(Concat(calldata_bv)))
```

Comment:
`Concat` will fail if the given list is a singleton. `concat` should be used.

Reply:
Concat, why are you so finicky? 🙈

Reply:
in c430f4e